Acast/Fastfile

```diff
@@ -4,19 +4,50 @@ default_platform :ios
 
 platform :ios do
   before_all do
+    if File.exist?("appleId")
```

Comment:
How about instead of doing this here, move this code to the `Appfile`, where it belongs to? :+1: